test_data/import_publications.py

```diff
@@ -132,58 +140,60 @@ def scan_resources():
 def delete_publications():
     resources = scan_resources()
     for resource in resources:
-        if resource['type']['S'] == 'Resource':
-            publication = resource['data']['M']
-            primary_partition_key = resource['PK0']['S']
-            primary_sort_key = resource['SK0']['S']
-            identifier = publication['identifier']['S']
-            owner = publication['owner']['S']
-            if 'test.no' in owner:
-                print(
-                    'Deleting {} - {}'.format(identifier, owner))
-                response = dynamodb_client.delete_item(
-                    TableName=publications_tablename,
-                    Key={
-                        'PK0': {
-                            'S': primary_partition_key
-                        },
-                        'SK0': {
-                            'S': primary_sort_key
-                        }
-                    })
+        publication = resource['data']['M']
+        primary_partition_key = resource['PK0']['S']
+        primary_sort_key = resource['SK0']['S']
+        identifier = publication['identifier']['S']
+        owner = publication['owner']['S']

```

Comment:
veldig kryptiske variabler her med 'M', 'S', osv. kan de dras ut til egne konstanter, eller er det vanskelig å gi de gode og beskrivende navn?

Reply:
'M' og 'S' kommer fra aws, 'S' står for String, 'M' for Map. Har ikke fått kommentar på det tidligere siden de som har sett på python-koden jobber med de samme json-strukturene.